### ISSUE TITLE
add support for /m/multiURL widgets on the dashboard

### ISSUE THREAD

**Issue by honestbleeps** [honestbleeps/Reddit-Enhancement-Suite#701]
http://www.reddit.com/r/Enhancement/comments/1u9kih/feature_request_multireddits_widgets_in_dashboard/

**Comment by erikdesjardins** [comment]
Duplicates #555, but this one has more information.

**Comment by ooker777** [comment]
4 years have passed. Is there anything new?

**Comment by honestbleeps** [comment]
nope, lots of priorities to juggle. perhaps you could consider contributing a pull request to add the feature?
